src/Microsoft.PowerShell.Commands.Utility/commands/utility/Get-Error.cs

```diff
@@ -75,11 +76,27 @@ protected override void ProcessRecord()
             foreach (object errorRecord in errorRecords)
             {
                 PSObject obj = PSObject.AsPSObject(errorRecord);
-                obj.TypeNames.Insert(0, "PSExtendedError");
 
-                // Remove some types so they don't get rendered by those formats
-                obj.TypeNames.Remove("System.Management.Automation.ErrorRecord");
-                obj.TypeNames.Remove("System.Exception");
+                if (obj.TypeNames.Contains("System.Management.Automation.ErrorRecord"))
+                {
+                    if (!obj.TypeNames.Contains("System.Management.Automation.ErrorRecord#PSExtendedError"))
+                    {
+                        obj.TypeNames.Insert(0, "System.Management.Automation.ErrorRecord#PSExtendedError");
+
+                        // Need to remove so this rendering doesn't take precedence
+                        obj.TypeNames.Remove("System.Management.Automation.ErrorRecord");
```

Comment:
Why isn't this a problem for Get-Process where we add "fake" types too?

Reply:
Don't think this is an issue, but potentially it might be because the error types tend to take precedence in the formatters explicitly (same reason why you need to `$error | Format-List -Force` and without `-Force` it simply gives the standard error view)

Reply:
Correct.  There is a whole concept of "Out of Band" formatting which takes precedence.  This is why when you do a recursive directory and you encounter an error, that error is in red and not part of the table output.  However, the side effect of this is these typenames need to be removed otherwise they will be formatted before my formatter gets a chance.